sound/soc/soc-dapm.c

```diff
@@ -855,6 +855,7 @@ static int dapm_create_or_share_kcontrol(struct snd_soc_dapm_widget *w,
 			kcname_in_long_name = true;
 		} else {
 			switch (w->id) {
+			case snd_soc_dapm_siggen:
```

Comment:
as discussed last week, I wonder if we should also add a snd_soc_dapm_sig_sink (the dual of a generator). This would be *very* useful for testing e.g. by letting us set-up all sort of DMA channels with a self-test checking.

Reply:
Sure, I will add it to my next update.